src/js/_enqueues/admin/common.js

```diff
@@ -1310,10 +1310,21 @@ $( function() {
 		$document.trigger( 'wp-notice-added' );
 	};
 
+	/**
+	 * Stores initial pagination value for comparison.
+	 */
```

Comment:
Imho this should just be a normal comment, not a doc like comment.

```suggestion
	// Stores initial pagination value for comparison.
```